app/classifier/drawing-tools/point-grid.jsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+const React = require('react');
+const createReactClass = require('create-react-class');
+const DrawingToolRoot = require('./root');
+const deleteIfOutOfBounds = require('./delete-if-out-of-bounds');
+const isInBounds = require('../../lib/is-in-bounds');
+const DeleteButton = require('./delete-button');
+
+const DELETE_BUTTON_ANGLE = 45;
+const SELECTED_RADIUS = {
+  large: 20,
+  small: 10
+};
+
+module.exports = createReactClass({
```

Comment:
This should extend `React.Component` rather than using `createReactClass`.

Reply:
Ah, my mistake, I was mimicking how the other drawing tools were written, and then put it through decaffeinate. I think I converted it over properly now.